common/version.go

```diff
@@ -1,6 +1,6 @@
 package common
 
-const AzcopyVersion = "10.3.0"
+const AzcopyVersion = "10.3.0-clfsload-preview"
```

Comment:
Consider making this an extension user-agent like the others, since the added clfsload functionality CURRENTLY doesn't goof with any azcopy functionality.